kde-frameworks/breeze-icons/metadata.xml

```diff
@@ -8,6 +8,9 @@
 	<upstream>
 		<bugs-to>https://bugs.kde.org/</bugs-to>
 	</upstream>
+	<use>
+		<flag name="24px">Install additional 24x24 icons</flag>
```

Comment:
Again, how shall the user make sense of this? What is the impact? What is lost?

I don't see the point of this micro optimisation.